### ISSUE TITLE
Great job!

### ISSUE THREAD

**Issue by compbcn** [kwstasg/UnrealVR#1]
Just to say Thank you! =)

Keep going. We are so many who love unreal, old unreal in new VR! 

I will be awesome!

THANK YOU!

**Comment by Ghostpepper123** [comment]
Agreed, was planning on making a VR decoupled version of Unreal 1998 when I discovered this project.  Super awesome stuff, I hope I can contribute something
